cypress/helpers/images.js

```diff
@@ -3,10 +3,10 @@ export function isSameImageAs(subject, referenceImage) {
   cy.wrap(subject)
     .should(([img]) => expect(img.complete).to.be.true)
     .then(([img]) => cy.request({ url: img.src, encoding: "base64" }))
-    .then((response) => cy.task("sha256", response.body))
+    .then((response) => cy.task("md5", response.body))
```

Comment:
i'd be careful with this and either use functions from the code itself to compute the hash or try to get as close as possible.

Reply:
Do you mean we should directly call something like https://github.com/prismicio/slice-machine/blob/165337d4da2f80238c156480f73a64016b07f7a7/packages/core/src/utils/str.ts#L15-L17 ?
tbh I wish Cypress offered a better way to compare an image to a local one, to avoid having to define these.

Reply:
Yeah that's what I meant, using what the code actually uses to test the same thing.
Or getting closer as possible (i thought the code was using sha256

Reply:
👍 I'll revert that commit

Reply:
done